### PR TITLE
Nodejs 13 compat and eslint/prettier glob fix

### PR DESCRIPTION
**Pull request by DamareYoh**
#### Is this adding or improving a _feature_ or fixing a _bug_?
Bugs

#### How does this change work?

* renamed common js files to have .cjs extension. 
* Removed singlequotes from glob inputs for eslint and prettier lint commands.

#### Have you checked that...?

- [x] The new code matches the existing patterns and styles.
- [x] The tests pass with `yarn test`.
- [x] The linter passes with `yarn lint`. (Fix errors with `yarn fix`.)
- [x] The relevant examples still work. (Run examples with `yarn start`.)

#### Does this fix any issues or need any specific reviewers?

Fixes: #
Reviewers: @ianstormtaylor

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
     "fix:eslint": "yarn lint:eslint --fix",
     "fix:prettier": "yarn lint:prettier --write",
     "lint": "yarn lint:eslint && yarn lint:prettier",
```
Comment (DamareYoh) [inline package.json]: i found that eslint and prettier were not finding files when i was running it locally, remove the singlequotes and this works.

**Comment by ianstormtaylor** [comment]
@DamareYoh does this still work in Node 12?

**Comment by DamareYoh** [comment]
@ianstormtaylor  sure does! I did this upgrade using node 12.13.1 and then upgraded to node 13.2.0 to double check it is working on that version.

**Comment by ianstormtaylor** [comment]
@DamareYoh beautiful, thanks!

**Pull request merged**